core/selection.js

```diff
@@ -156,6 +157,21 @@ class Selection {
     }
   }
 
+  isReversed() {
```

Comment:
Don't think this is necessary. Take a look at normalizedToRange. There looks to be a much easier way to figure reversal status.